addItemsScript.js

```diff
@@ -1,188 +1,140 @@
-            imageUrl: '/images/owl.jpg',
-            filePath: null,
-            category: 'character options'
-        }
-    }).then(([item, created]) => {
-        if (created) {
-            console.log("Item added!");
-        } else {
-            console.log("Item already exists, not added!");
-        }
-    }).catch((error) => {
-        console.error("Error handling item:", error);
-    });
+    async function addItem(itemData) {
+        try {
+            const [item, created] = await Item.findOrCreate({
+                where: { itemNumber: itemData.itemNumber },
+                defaults: itemData.defaults
+            });
+            if (created) {
+                console.log(`Item ${itemData.itemNumber} added!`);
+            } else {
+                console.log(`Item ${itemData.itemNumber} already exists, not added!`);
+            }
+        } catch (error) {
+            console.error("Error handling item:", error);
+        }
+    }
 
-    Item.findOrCreate({
-        where: { itemNumber: '002' },
-        defaults: {
-            name: 'Companion Animals Expanded (5e)',
-            price: 0.00,
-            authorName: 'Jane Doe',
-            authorWebsite: 'http://google.com',
-            description: 'Embark on a thrilling journey through uncharted realms with our groundbreaking Dungeons & Dragons 5th Edition expansion: "Companion Animals Expanded". Delve into the depths of mystique as you encounter an eclectic array of unique and peculiar companion animals, each brimming with newfound actions and abilities to enrich your gameplay experience. Our expansion boasts meticulously crafted content, honed through rigorous playtesting, ensuring seamless integration into your adventures. From the enigmatic Shadow Wisp to the majestic Stormcaller Gryphon, discover companions that defy convention and breathe new life into your campaigns. Unleash the power of the arcane, forge unbreakable bonds, and chart your destiny.',
-            imageUrl: '/images/rat.jpg',
-            filePath: null,
-            category: 'rule books'
-        }
-    }).then(([item, created]) => {
-        if (created) {
-            console.log("Item added!");
-        } else {
-            console.log("Item already exists, not added!");
-        }
-    }).catch((error) => {
-        console.error("Error handling item:", error);
-    });
+    const itemsToAdd = [
+        {
+            itemNumber: '001',
+            defaults: {
+                name: 'The Owlfolk Expansion: Wings of Winter',
+                price: 9.99,
+                authorName: 'Jane Doe',
+                authorWebsite: 'http://google.com',
```

Comment:
do we want author websites? All author websites should just be the path to their username, right?

Reply:
Actually, this can just be changed later when we have more routes for creators.